control-plane/pkg/reconciler/broker/broker.go

```diff
@@ -212,6 +213,20 @@ func (r *Reconciler) reconcileKind(ctx context.Context, broker *eventing.Broker)
 	return statusConditionManager.Reconciled()
 }
 
+func resolveTopicName(broker *eventing.Broker) string {
+	topicName := kafka.Topic(TopicPrefix, broker)
+	topicAnnotationValue, ok := isCustomTopic(broker)
+	if ok {
+		topicName = topicAnnotationValue
+	}
+	return topicName
+}
+
+func isCustomTopic(broker *eventing.Broker) (string, bool) {
+	topicAnnotationValue, ok := broker.Annotations[topicAnnotation]
```

Comment:
Maybe just return `bool` since the function name starts with `is`?

Reply:
yeah, this is not ready. I am just adding to my WIP